modules/javafx.graphics/src/main/java/com/sun/javafx/scene/text/TextLayout.java

```diff
@@ -199,7 +199,7 @@ public Hit(int charIndex, int insertionIndex, boolean leading) {
      */
     public boolean setTabSize(int spaces);
 
-    public Hit getHitInfo(float x, float y);
+    public Hit getHitInfo(float x, float y, String text);
```

Comment:
Perhaps we could add javadoc with an explanation for the 'text' parameter, that it's expected to be null in the case of TextFlow and non-null in the case of Text.

Reply:
Added javadoc comment